app/controllers/api/v1/api_controller.rb

```diff
@@ -6,6 +6,12 @@ class ApiController < ApplicationController
 
     private
 
+    def respond_with(serializer)
+      respond_to do |format|
+        format.json { render json: serializer }
+      end
+    end
+
```

Comment:
I think re-implementing `respond_with` in this way may cause problems for future maintenance - new developers on the project may think it's the standard Rails feature. We should call it something else or just use `respond_to` within the action directly - that's the recommended pattern.

Alternatively we could use the responders gem :smile: